apps/designer/components/elements/Scene.tsx

```diff
@@ -0,0 +1,44 @@
+import SceneComp from "./SceneComp";
+import { createCamera } from "./freeCamera";
+import { createGround } from "./ground";
+import { createSkyBox } from "./skybox";
+import { createFog } from "./fog";
+import { createModel } from "./models";
+import * as BABYLON from "@babylonjs/core";
+import { HemisphericLight, Vector3 } from "@babylonjs/core";
+import "@babylonjs/loaders/glTF";
+
+//Work under progress, the demo component contains the code that will use Zod and React Custom Hooks in order to render the scene. This will be a part of the code later on when the work with Zod is complete
```

Comment:
is this needed? Use the format "// TODO: ...." to add semantic comments that can be recognized by code editors